src/test/java/org/folio/permstest/RestVerticleTest.java

```diff
@@ -617,9 +620,68 @@ public void testPurgeDeprecatedPerm(TestContext context) {
     // verify perm user
     response = send(HttpMethod.GET, "/perms/users/" + permUserId, null, context);
     permUsersObject = response.body;
-    context.assertTrue( permUsersObject.getJsonArray("permissions").contains(perm1));
-    context.assertFalse( permUsersObject.getJsonArray("permissions").contains(perm2)); // perm2 is gone
-    
+    context.assertTrue(permUsersObject.getJsonArray("permissions").contains(perm1));
+    context.assertFalse(permUsersObject.getJsonArray("permissions").contains(perm2)); // perm2 is gone
+  }
+
+  @Test
+  public void testPermissionNameReplace(TestContext context) {
+    // clean up deprecated perms first
+    Response response = send(HttpMethod.POST, "/perms/purge-deprecated", null, context);
+
+    // seed modA with perm1 and perm2
+    String perm1 = "permA" + UUID.randomUUID().toString();
+    String perm2 = "permB" + UUID.randomUUID().toString();
+    JsonObject permissionSet = new JsonObject()
+        .put("moduleId","moduleTestReplaceA-1.0.0")
```

Comment:
Can we rename the modules to moduleTestReplaceX and moduleTestReplaceY? A and B are already used for permission names.

Reply:
Renamed.